index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Microverse Portfolio project</title>
+  </head>
+  <body>
+    <header>
+      <div class="Logo">
+        <h4><a href="#"></a>Solomon Afutu</h4>
+        <nav>
+          <ul class="Menu-bar">
+            <li class="menu-line"></li>
+            <li class="menu-line"></li>
+            <li class="menu-line"></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <section class="Header-main">
+      <h1>
+        I’m Solomon<br />
+        Glad to see <br />
+        you!
+      </h1>
+      <p>
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <h3>LET'S CONNECT</h3>
+      <ul>
+        <a href="https://twitter.com/SolomonAfutu"
+          ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
+        ></a>
+        <a href="https://www.linkedin.com/in/solomon-afutu-164b95255/"
+          ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
+        ></a>
+        <a href="https://github.com/SolomonTheCoder"
+          ><i class="fa fa-github fa-2x" aria-hidden="true"></i
+        ></a>
+        <i class="fa fa-medium fa-2x" aria-hidden="true"></i>
+        <i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i>
+      </ul>
```

Comment:
- [ ] As suggested by the previous reviewer you should make sure that thepositioning of the social icons should be exactly how its mentioned in the figma template. Please reduce the margin of social media icons from its left to make it look identical to the fiigma design template 

**Margin of social icons on your app**

<img width="513" alt="Screenshot 2023-02-15 at 4 41 11 AM" src="https://user-images.githubusercontent.com/44798044/218888717-07ef8b20-16ed-4597-9b8a-d48106c1e24e.png">

**Margin of social  icons on Figma template**

<img width="518" alt="Screenshot 2023-02-15 at 4 39 38 AM" src="https://user-images.githubusercontent.com/44798044/218888882-c78e8168-464b-4030-b8ab-912bf875d03e.png">

Reply:
- [ ] Please implement this change.  The Twitter logo and the `LET'S CONNECT` text should have the same spacing from the left as clearly shown by the previous code reviewer. 😄